lib/main.dart

```diff
@@ -217,34 +177,46 @@ class _MyHomePageState extends State<MyHomePage> {
   Widget build(BuildContext context) {
     return Scaffold(
       body: Center(
-        child: Column(
-          mainAxisAlignment: MainAxisAlignment.center,
-          children: [
-            IconButton(
-              icon: Image.asset("assets/images/familymart.png"),
-              iconSize: 128.0,
-              onPressed: () async {
-                final pay = await widget.repo.getRecommendedPay("family_mart");
-                launchPay(pay);
-              },
-            ),
-            IconButton(
-              icon: Image.asset("assets/images/lawson.png"),
-              iconSize: 128.0,
-              onPressed: () async {
-                final pay = await widget.repo.getRecommendedPay("lawson");
-                launchPay(pay);
-              },
-            ),
-            IconButton(
-              icon: Image.asset("assets/images/seveneleven.png"),
-              iconSize: 128.0,
-              onPressed: () async {
-                final pay = await widget.repo.getRecommendedPay("seven_eleven");
-                launchPay(pay);
+        child: Builder(
+          builder: (context) {
+            final storeRepo = StoreRepository();
+            return FutureBuilder<List<Store>>(
+              future: storeRepo.getStores(),
+              builder: (context, snap) {
+                if (!snap.hasData) {
+                  return Container();
+                } else {
```

Comment:
IMOなんだけど、インデントを深くすると可読性が下がるので、
```suggestion
                if (!snap.hasData) {
                    return Container():
                final stores = snap.data;
                ...
```
はどうですか？